Gemfile

```diff
@@ -54,7 +54,7 @@ end
 group :development do
   gem 'quiet_assets'
   gem 'better_errors'
-  # gem 'binding_of_caller'
+  gem 'binding_of_caller'
```

Comment:
@roosdebildt just add the comment again, re-bundle and make another commit. It's cool to have all the intermediary commits in this PR. RGSoC is about learning, it would make no sense if we didn't accept commits that bounce back and force or revert an "oops" ;)